shared/schemas/workflow-options-output-schema.json

```diff
@@ -0,0 +1,48 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Workflow Options",
+    "description": "Workflow options",
+    "type": "object",
+    "properties": {
+        "status": {
+            "description": "The current status of a workflow execution",
+            "$ref": "workflow-status-schema.json"
+        },
+        "currentVersion": {
```

Comment:
Do we still need following fields?
I would keep them for backward compatibility to simplify existing workflows' adaptation to the new schema.

Reply:
in my opinion it is ok to remove it